tensorboard/components/tf_dashboard_common/dashboard-style.html

```diff
@@ -54,8 +54,9 @@
       }
 
       .sidebar-section {
-        border-top: solid 1px rgba(0, 0, 0, .12);
-        padding: var(--sidebar-vertical-padding) 0 var(--sidebar-vertical-padding) var(--sidebar-left-padding);
+        border-top: solid 1px rgba(0, 0, 0, 0.12);
```

Comment:
This change violates: https://google.github.io/styleguide/htmlcssguide.html#Leading_0s
Honestly, I think it is job of modernizer to get rid of "0" for shaving few bytes off of CSS payload so I am okay with whatever if it is hard to configure the prettier.

Reply:
Agreed; the leading 0 makes it easier to read, minifcation can be done
by a compiler, and Prettier has no configuration option here, so I’ll
leave this alone.